src/Containers/SearchFilterContainer.php

```diff
@@ -19,12 +21,20 @@ public function call(Twig $twig, SearchService $searchService): string
         }
 
         $searchResults = $searchService->getResults();
+        $currentCategory = $searchService->getCategoryService()->getCurrentCategory();
+        $showCategoryFilter = true;
+
+        if (null !== $currentCategory && $currentCategory->level !== 1) {
```

Comment:
Please don't use [yoda conditions](https://en.wikipedia.org/wiki/Yoda_conditions).

```suggestion
        if ($currentCategory !== null && $currentCategory->level !== 1) {
```

Reply:
Maybe you wanna change the condition to `> 1` to make it more clear that everything above should not be shown. Also a comment might be helpful, otherwise someone reading this may be confused, why the category filter is not shown further levels below.